### PR TITLE
feat(ui): Add tooltip to Member row

### PR DESCRIPTION
**Pull request by billyvg**
Adds a new <Tooltip> component (basically does the same thing as the mixin, but as a component).

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: docs-ui/components/tooltip.stories.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+import {withInfo} from '@storybook/addon-info';
+// import {action} from '@storybook/addon-actions';
```
Comment (MaxBittker) [inline docs-ui/components/tooltip.stories.js]: random commented line

**Review by MaxBittker — APPROVED** [review]
good stuff :) love components. Are there any drawbacks to using this vs the mixin? or should we move over entirely someday?

**Comment by MaxBittker** [comment]
not actually sure what failed in travis?

**Comment by billyvg** [comment]
No drawbacks, idea is to 1) get away from mixins 2) use components so it'll be easier to swap out 
 bootstrap if necessary.

**Pull request merged**